docs/sponsorship/slack.rst

```diff
@@ -0,0 +1,75 @@
+Slack Network Sponsorship
+=========================
+
+Write the Docs is a community focused on all things related to software documentation.
+Our :doc:`Slack network </slack>` is the beating heart of our community,
+where thousands of documentarians hang out regularly to chat.
+Sponsorship of our Slack network does two things:
+
+* It allows us to cover the cost of moderating and managing our Slack community
+* It allows you to get your name and message in front of our audience of :doc:`/documentarians`
+
+You can join our Slack and read some of the messages to get a feel for the community.
+
+Audience
+--------
+
+Our audience is made up of technical writers and decision makers in the software documentation industry.
+The current Slack stats are:
+
+* Over 17,000 users
+* 100+ channels
+
+We don't have specific demographic data for our Slack,
+but our conferences are made up of:
+
+- Technical Writers (60%)
+- Developers (10%)
+- Support Staff (10%)
+- Managers (10%)
+- Community Contributors, Enthusiasts & Other Folks (10%)
+
+
+Sponsored entry format
+----------------------
+
+Sponsor copy will be posted to the ``#announcements`` channel in Slack,
+which is joined by every user in our Slack.
+It's used for community-wide announcements,
+so your message will be very visible in the community.
+
+The current format is:
+
+* Up to 300 characters of copy
+* Up to 2 links in that copy
+  - These links will be auto-expanded into a rich embed in Slack.
+
+We're still experimenting with exact format details,
+so let us know if you have any questions.
+
```

Comment:
Should we note that Slack rules generally still apply, outside of the "No Sales-Y Content" rule since this is a sponsored post? Spitballing.

```suggestion

Content Guidance
------------------------

The provided copy should still respect our general [Slack Rules](https://www.writethedocs.org/slack/) and [Code of Conduct](https://www.writethedocs.org/code-of-conduct/). All other community engagement by your employees, contractors, or other affiliates *must* respect the general Slack rules. 

With regards to the Slack No Sales-Y Content rule, sponsored posts have an exemption to the *No Direct Pitches*, *No links without context*, and *No Brands* subrules.  We would still ask that you respect the *Be Transparent* rule, as it is likely to improve the reception of your copy.

Write the Docs reserves the right to request changes to or reject copy that violates the Slack Rules or Code of Conduct.


```

Reply:
Yea, this is a great section to add! 👍